src/Majora/Framework/Loader/Bridge/Form/Type/EntityCollectionType.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Majora\Framework\Loader\Bridge\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Majora\Framework\Loader\LoaderCollection;
+use Majora\Framework\Loader\Bridge\Form\DataTransformer\EntityCollectionTransformer;
+
+/**
+ * Class EntityCollectionType
+ */
+class EntityCollectionType extends AbstractType
+{
+    /**
+     * @var LoaderCollection
+     */
+    private $loaderCollection;
+
+    /**
+     * EntityCollectionType constructor.
+     *
+     * @param LoaderCollection $loaderCollection
+     */
+    public function __construct(LoaderCollection $loaderCollection)
+    {
+        $this->loaderCollection = $loaderCollection;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+    
+        $builder->addModelTransformer(new EntityCollectionTransformer($this->loaderCollection->get($options['loader_alias'])));
+    }
+
+    /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $self = $this;
```

Comment:
You dont need to do this since php 5.4, `$this` can be used into closures.
MajoraFrameworkExtraBundle supports Php > 5.5.9 (like Symfony) so no problem.